sidecar-injector/README.md

```diff
@@ -126,61 +242,272 @@ For this section, you'll work from a test namespace (test-namespace). Later you
     EOL
     ```
 
-4. Deploy an app in Kubernetes cluster, take `sleep` app as an example
+4. Deploy an app with the Secretless Sidecar, take `test-app` app as an example
     ```bash
     ~$ cat << EOF | kubectl -n test-namespace create -f -
     apiVersion: extensions/v1beta1
     kind: Deployment
     metadata:
-      name: sleep
+      name: test-app
     spec:
       replicas: 1
       template:
         metadata:
           annotations:
-            sidecar-injector.secretless.io/inject: "yes"
-            sidecar-injector.secretless.io/config: "sleep-secretless-config"
+            sidecar-injector.cyberark.com/inject: "yes"
+            sidecar-injector.cyberark.com/secretlessConfig: "test-secretless-config"
+            sidecar-injector.cyberark.com/injectType: "secretless"
           labels:
-            app: sleep
+            app: test-app
         spec:
           containers:
-          - name: sleep
-            image: everpeace/curl-jq
-            command: ["/bin/sleep","infinity"]
+            - name: app
+              env:
+                - name: http_proxy
+                  value: "http://0.0.0.0:3000"
+              image: googlecontainer/echoserver:1.1
     EOF
     ```
 
-5. Verify sidecar container injected
+5. Verify Secretless sidecar container injected
     ```bash
     ~$ kubectl -n test-namespace get pods
     ```
     ```
     NAME                     READY     STATUS        RESTARTS   AGE
-    sleep-5c55f85f5c-tn2cs   2/2       Running       0          1m
+    test-app-5c55f85f5c-tn2cs   2/2       Running       0          1m
     ```
 
 6. Test Secretless
     ```bash
-    ~$ a_sleep_pod=$(kubectl \
+    ~$ a_test_pod=$(kubectl \
```

Comment:
Would be good to add to these two commands a comment line about what they're testing